### ISSUE TITLE
yarn upgrade fails

### ISSUE THREAD

**Issue by stuta** [nostalgic-css/NES.css#392]
**Describe the bug**
yarn upgrade fails

nes.css@2.3.0: The engine "node" is incompatible with this module. Expected version "10.x". Got "13.5.0"

**To Reproduce**
yarn upgrade-interactive --latest

**Comment by BcRikko** [comment]
ref #379

**Comment by stuta** [comment]
I think you should change package.json:

 "engines": {
    "node": ">=10.0.0"
  },

**Comment by BcRikko** [comment]
We received an [issue](#379) that says it would cause an error in version 13, so I think `"node": ">=10.0.0 <13.0.0"` is better 👍

**Comment by stuta** [comment]
I can't see what is the setup error in #379. I had no setup errors with Node 13 before this engine change.

And if there is some error then instead of making this unusable with node versions 13.0 - 13.5 maybe the right solution is to make it compatible with 13?

**Comment by BcRikko** [comment]
Sorry, I didn't try to install with node 13.
If there are no errors with node 13, I agree to the comment you have suggested. 👍

**Comment by virtuoushub** [comment]
**tl;dr** try `yarn install --ignore-engines`

_longer comment_

Using node v12.14.0 (npm v6.13.4), I was able to build the project as it is in the master branch.

Using node v13.5.0 (npm v6.13.4) I ran into the following issue while building this project as it is in the master branch. The issue is `node-sass` related [ see <https://github.com/sass/node-sass/releases/tag/v4.12.0> ]):

```
...
> node-sass --output-style expanded --source-map true --functions scripts/scssFunctions.js scss/nes.scss css/nes.css

/foo/NES.css/node_modules/node-sass/lib/binding.js:13
      throw new Error(errors.unsupportedEnvironment());
      ^

Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (79)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.12.0
...
```

~~In order~~ One way to get NES.css to build under Node 13, is to upgrade `node-sass`to `4.13.0` [ see <https://github.com/sass/node-sass/releases/tag/v4.13.0> ].

> "node": ">=10.0.0 <13.0.0"

Will work for how the project is now, where as something like `"node": ">=10.0.0 <14.0.0"` will work if `"node-sass"` gets upgraded to `"^4.13.0"`.

**edited for clarity and added tl;dr**
